pages/book/cells.mdx

```diff
@@ -47,11 +47,34 @@ Every cell, being a [quadtree][quadtree], has an attribute called _level_, which
 
 [Exotic](#cells-kinds) cells have different rules for determining their level, which are described [on this page in TON Docs](https://docs.ton.org/develop/data-formats/exotic-cells).
 
-### Standard representation [#cells-representation]
+### Serialization [#cells-serialization]
+
+Before a cell can be transferred over the network or stored on disk, it must be serialized. There are several common formats, such as [standard `Cell{:tact}` representation](#cells-representation) and [BoC](#cells-boc).
+
+#### Standard representation [#cells-representation]
+
+Standard [`Cell{:tact}`](#cells) representation is a common serialization format for cells first described in the [tvm.pdf](https://docs.ton.org/tvm.pdf). Its algorithm representing cells in octet (byte) sequences begins with serializing the first $2$ bytes called descriptors:
+
+* _Refs descriptor_ is calculated according to this formula: $r + 8 * k + 32 * l$, where $r$ is the number of references contained in the cell (between $0$ and $4$), $k$ is a flag for the cell kind ($0$ for [ordinary](#cells-kinds) and $1$ for [exotic](#cells-kinds)), and $l$ is the [level](#cells-levels) of the cell (between $0$ and $3$).
+* _Bits descriptor_ is calculated according to this formula $\lfloor\frac{b}{8}\rfloor + \lceil\frac{b}{8}\rceil$, where $b$ is the number of bits in the cell (between $0$ and $1023$).
+
+Then, the data bits of the cell themselves are serialized as $\lceil\frac{b}{8}\rceil$ $8$-bit octets (bytes). If $b$ is not a multiple of eight, a binary $1$ and up to six binary $0$s are appended to the data bits.
+
+Next, the $2$ bytes store the depth of the refs, i.e. their [levels](#cells-levels).
+
+Finally, for every reference cell we store its [SHA-256][sha-2] hash of its standard representation in $32$ bytes per each such cell, recursively repeating the algorithm laid out over and over. Notice, that cyclic cell references are not allowed, which means this recursion always ends in a well-defined manner.
+
+If we were to compute the hash of the standard representation of this cell, all the bytes from steps above would be concatenated together and then hashed using [SHA-256][sha-2] hash.
```

Comment:
Maybe we should also mention here that this is exactly what happens under the hood of TVM when the `c.hash()` method is called?

Reply:
Do you mean the `HASHCU` instruction?

Reply:
Yes, and `HASHSU` too. I meant that we can reference https://docs.tact-lang.org/ref/core-cells#cellhash here. But maybe it's not necessary here since there's a  reference in opposite direction from `c.hash()` to this page.